packages/venia-concept/src/components/Checkout/paymentsForm.js

```diff
@@ -22,228 +21,229 @@ const DEFAULT_FORM_VALUES = {
     addresses_same: true
 };
 
-class PaymentsForm extends Component {
-    static propTypes = {
-        cancel: func.isRequired,
-        classes: shape({
-            address_check: string,
-            body: string,
-            button: string,
-            braintree: string,
-            city: string,
-            footer: string,
-            heading: string,
-            postcode: string,
-            region_code: string,
-            street0: string,
-            textInput: string
-        }),
-        initialValues: shape({
-            city: string,
-            postcode: string,
-            region_code: string,
-            sameAsShippingAddress: bool,
-            street0: array
-        }),
-        submit: func.isRequired,
-        submitting: bool,
-        countries: array
-    };
-
-    static defaultProps = {
-        initialValues: {}
-    };
-
-    state = {
-        isRequestingPaymentNonce: false
-    };
-
-    render() {
-        const { classes, initialValues } = this.props;
-        const { formChildren } = this;
+const BillingAddressFields = ({ classes, countries }) => {
```

Comment:
We definitely want to stick to one component definition per file. There's nothing wrong with having more files.